creation.rst

```diff
@@ -0,0 +1,60 @@
+Creating your wallet
+====================
+
+Choose a name
+-------------
+
+Some people use multiple wallets!  Just make sure it's very clear to avoid confusion!
+
+.. image:: png/creation_namewallet.png
+
+Choose the wallet type
+----------------------
+
+Electrum supports some advanced features.  
+Let's stick with the default.
+
+.. image:: png/creation_wallettype.png
+
+
+Choose how keys are stored
+--------------------------
+
+Electrum supports some advanced features.
```

Comment:
It might make more sense to explain those features somewhere else and link to it here.